I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


The label _call_freeTime is defined twice, at File "game/freetime.rpy", line 50:
    call freeTime from _call_freeTime
and File "game/freetime-events.rpy", line 705:
    call freeTime from _call_freeTime


Ren'Py Version: Ren'Py 8.0.3.22090809
Sat Dec  3 10:31:54 2022
